api/coreservice.go

```diff
@@ -441,6 +446,19 @@ func (core *coreService) validateChainID(chainID uint32) error {
 	return nil
 }
 
+func (core *coreService) validateWeb3Rewarding(selp action.SealedEnvelope) error {
+	if ge := core.bc.Genesis(); ge.IsToBeEnabled(core.bc.TipHeight()) || selp.Encoding() != uint32(iotextypes.Encoding_ETHEREUM_RLP) {
+		return nil
+	}
+	switch selp.Action().(type) {
+	case *action.ClaimFromRewardingFund,
```

Comment:
do i recall correctly, that there's 2 new claim actions enabled by web3 format?
if that's the case, need to check 2 possible types?